src/scss/_mixins.scss

```diff
@@ -145,7 +145,7 @@
 		}
 
 		@font-face {
-			src: url($o-fonts-path + '#{$family}-#{$font-suffix}' + '.woff');
```

Comment:
Not sure if it's important but should we consider including `format('woff2')` etc in these?